### ISSUE TITLE
Quasar Vue/Cli plug-in does not work

### ISSUE THREAD

**Issue by smolinari** [quasarframework/quasar#12944]
### Discussed in https://github.com/quasarframework/quasar/discussions/12943

<div type='discussions-op-text'>

<sup>Originally posted by **murugappanrm** March 26, 2022</sup>
I using vue/cli version 5.0.1 and quasar/cli version 1.3.2. I install quasar using the vue/cli option. The install was successful but when i run "npm run serve" the page created is messed-up: 

![Screenshot1](https://user-images.githubusercontent.com/63133893/160226933-be5c9607-d3a2-4b57-821f-715f7bacc01d.png)
</div>

Quasar info: 

```
Operating System   	Linux(5.13.0-35-generic) - linux/x64
NodeJs             	16.14.0

Global packages    	
  NPM              	8.5.2
  yarn             	1.22.17
  @quasar/cli      	1.3.2
  @quasar/icongenie	Not installed
  cordova          	Not installed

Networking         	
  Host             	scotts-ubuntu
  wlxbcec23c3706e  	192.168.178.67

```

Vue CLI version: 5.0.4

Install the plug-in with the defaults and you get the problem noted in the discussion. I was also using yarn.

Scott

**Comment by yusufkandemir** [comment]
Closing in favor of https://github.com/quasarframework/vue-cli-plugin-quasar/issues/52

**Comment by smolinari** [comment]
@yusufkandemir - Thanks Yusuf. 

Scott

**Comment by rstoenescu** [comment]
Released vue-cli-plugin-quasar v5.0.1 which has support for Vue CLI v5. https://github.com/quasarframework/vue-cli-plugin-quasar/releases/tag/v5.0.1
